### ISSUE TITLE
"Space" is enough to trigger the input selection method

### ISSUE THREAD

**Issue by teto** [fcitx/fcitx#361]
I've tried to configure the trigger with different combinations Ctrl+space/Super+space but everytime I type <space>, it triggers the input method selection popup which basically makes fcitx useless.
If I type Super+Space then a space is inputed into my terminal (termite). I would expect the opposite :/

i checked for https://askubuntu.com/questions/736638/fcitx-wont-trigger-ime-on-superspace but it's not that. I am on a macbook so maybe that's why ? Where is the part of the code involved I could check ?

My config is basically the same as https://github.com/fcitx/fcitx/issues/357 (for which I hope to send my fix soon).

**Comment by wengxt** [comment]
... what's that left + space?
Fcitx only supports key combination with modifier keys.

**Comment by teto** [comment]
I meant left Super sorry. I've tried removing the fcitx config to get a new one but same problem.
Here I typed just space while Ctrl+Space is setup as the trigger but the popup arrives: 

![2017-09-19-013818_1280x800_scrot](https://user-images.githubusercontent.com/886074/30553186-5df7ce8e-9cdb-11e7-8c6b-e4800d846d32.png)

I've put the diagnose just in case (the crash log was due to restart failing on nixos because fcitx is wrapped, it's not important).
[log.txt](https://github.com/fcitx/fcitx/files/1311464/log.txt)
